src/charm.py

```diff
@@ -812,14 +825,14 @@ def _addon_reconciliation(self, addon_type):
         install_addons = desired_addons - current_installed_addons
         uninstall_addons = current_installed_addons - desired_addons
         for addon in install_addons:
-            logger.debug("Install %s: %s", addon_type, repr(addon))
+            logger.info("Install %s: %s", addon_type, repr(addon))
             result = self._wp_addon_install(addon_type=addon_type, addon_name=addon)
             if not result.success:
                 raise exceptions.WordPressBlockedStatusException(
                     f"failed to install {addon_type} {repr(addon)}"
                 )
         for addon in uninstall_addons:
```

Comment:
Wouldn't it be convenient to execute the two for loops on a separate method?

Reply:
It's just 13 lines of code for now, but I will definitely move that to a new method if there are more features added to these loops.

Reply:
Or do you mean that one method can handle both the installing and uninstalling logic to reduce duplicate code? I considered that and I actually did the same thing for `_addon_reconciliation` which combines the `_theme_reconciliation` and `_plugin_reconciliation`. But this two for-loops is not very long, so I didn't do that. As before, I will definitely that if more features are added to these loops.